### PR TITLE
Update to v 8.1.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- version bumped up and sha256 updated
 - script command modified to include --no-build-isolation
 - dependencies revised and updated
 - tests modified

### CODE REVIEW

File: recipe/run_test.sh [hunk 1]
```diff
@@ -10,14 +10,14 @@ jupyter run -h
 
 if [ "$(uname)" == "Darwin" ]; then
     ulimit -n 4096
-    pytest --asyncio-mode=auto --pyargs jupyter_client --cov jupyter_client --cov-report term-missing:skip-covered --no-cov-on-fail -k "not (test_install_kernel_spec_prefix)" 
+    pytest tests --pyargs jupyter_client --cov jupyter_client --cov-report term-missing:skip-covered --no-cov-on-fail -k "not test_tcp_cinfo"
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/run_test.sh:13]: in one of my previous PR was told that we do not need to run test coverage, or shall we ?

Comment (akabanovs) [reply recipe/run_test.sh:13]: If it runs without problem and reaches the threshold, then why not?

Comment (akabanovs) [reply recipe/run_test.sh:13]: I think if it failed the coverage, then we could just ignore and deactivate it (or reduce threshold for us to have it passed) as it is up to the maintainers to maintain it.

Comment (M-Waszkiewicz-Anaconda) [reply recipe/run_test.sh:13]: yes, make sense

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**